remotetracker/tracker.go

```diff
@@ -0,0 +1,15 @@
+package remotetracker
+
+// Tracker represents a remote issue tracker
+// for e.g github, trello, jira, bugzilla
+type Tracker interface {
+	Fetch(query string) error
+	Import() error
```

Comment:
@aslakknutsen We are thinking to store the into a staging table (`tracker_items`). Later other part of the system can get data from here.